pkg/systemd/logs.js

```diff
@@ -192,7 +197,50 @@ $(function() {
             }
         }
 
+        function clear_service_list() {
+            $('#journal-services-list').empty()
+                    .append($('<li>').append($('<a>')
+                            .text(_("All"))
+                            .attr('data-service', "")))
+                    .append($('<li>')
+                            .addClass("divider"));
+        }
+
+        function load_service_filters(match, options) {
+            var service_options = Object.assign({ "output": "verbose" }, options);
+            var cmd = journal.build_cmd(match, service_options)[0].join(" ");
+            cmd += " | grep SYSLOG_IDENTIFIER= | sort -u";
+            cockpit.spawn(["bash", "-ec", cmd], { host: options.host, superuser: "try" })
```

Comment:
`sh` is enough here, doesn't need `bash`. This makes no difference on Fedora/RHEL, but is faster on Debian/Ubuntu whose default shell is `dash`.